.github/dependabot.yml

```diff
@@ -7,3 +8,13 @@ updates:
       # Check for updates to GitHub Actions every week on Saturday
       interval: "weekly"
       day: "saturday"
+  - package-ecosystem: "gomod"
+    directories:
+      - "src/cloud-api-adaptor"
+      - "src/cloud-api-adaptor/podvm"
+      - "src/cloud-providers"
+      - "src/csi-wrapper"
+      - "src/peerpod-ctrl"
+      - "src/webhook"
```

Comment:
Taking aside `vendor` there are still a few `go.mod` files. Do you think we should include them as well?

Reply:
I'm confused by this comment - what are the other go.mods. I don't think we even vendor in this repo?

Reply:
I simply `find -iname 'go.mod'` which showed more sub-projects than the listed ones, not sure whether they should be covered.

Reply:
Oh, they were left-behind dirs from some testing. You are right and this is the full set.